sources/packages/api/src-sql/SINValidations/Create-sin-validations.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE IF NOT EXISTS sims.sin_validations (
+  id SERIAL PRIMARY KEY,
+  date_sent TIMESTAMP WITH TIME ZONE,
+  date_received TIMESTAMP WITH TIME ZONE,
+  file_sent VARCHAR(50),
+  file_received VARCHAR(50),
+  given_name_sent VARCHAR(50),
+  surname_sent VARCHAR(50),
+  dob_sent Date,
+  request_status_code VARCHAR(2),
+  match_status_code VARCHAR(2),
+  sin_match_status_code VARCHAR(2),
+  surname_match_status_code VARCHAR(2),
+  given_name_match_status_code VARCHAR(2),
+  dob_match_status_code VARCHAR(2),
+  -- Audit columns
+  created_at timestamp without time zone NOT NULL DEFAULT now(),
```

Comment:
audit fields we decided to make `with timezone` right? @andrewsignori-aot @dheepak-aot @guru-aot @henryoforeh-aot

Reply:
missed it, will change